x-pack/plugins/maps/public/plugin.ts

```diff
@@ -106,14 +113,25 @@ export class MapsPlugin
       MapsPluginSetupDependencies,
       MapsPluginStartDependencies
     > {
+  readonly _initializerContext: PluginInitializerContext<MapsXPackConfig>;
+
+  constructor(initializerContext: PluginInitializerContext<MapsXPackConfig>) {
+    this._initializerContext = initializerContext;
+  }
+
   public setup(core: CoreSetup, plugins: MapsPluginSetupDependencies) {
+    const config = this._initializerContext.config.get<MapsConfigType>();
+    const kibanaVersion = this._initializerContext.env.packageInfo.version;
```

Comment:
Replace with core.injectedMetadata.getKibanaVersion();

Reply:
I made this change but actually ended up rolling it back. The method `getKibanaVersion` isn't available on `core.injectedMetadata` when called in an NP context. I did a quick search and it looks like `initializerContext.env.packageInfo.version` is a pretty common way to obtain the version in NP. I'm going to leave this as is for now. I would consider doing it the same way in legacy but I'd rather not touch since 1. it's working and 2. that's all going away in the next NP-related PR.